nvm.sh

```diff
@@ -3024,6 +3036,7 @@ nvm() {
         nvm_echo '  nvm ls-remote [<version>]                   List remote versions available for install, matching a given <version> if provided'
         nvm_echo '    --lts                                     When listing, only show LTS (long-term support) versions'
         nvm_echo '    --lts=<LTS name>                          When listing, only show versions for a specific LTS line'
+        nvm_echo '    --min=<version>                           When listing, only show versions greater than or equal to <version>, including minor/patch updates for installed versions'
```

Comment:
should this same option work for `nvm ls` as well?

Reply:
That would be an no-op since the installed versions, which is what reported by `nvm ls`, are always listed.